cockpit/cockpit-image-builder.spec

```diff
@@ -1,11 +1,14 @@
 Name:           cockpit-image-builder
-Version:        1
+Version:        54
 Release:        1%{?dist}
-Summary:        image builder frontend for use with Cockpit
+Summary:        Image builder plugin for Cockpit
 
-License:        Apache
+License:        Apache-2.0
 URL:            http://osbuild.org/
-Source0:        %{name}-%{version}.tar.gz
+Source0:        https://github.com/osbuild/image-builder-frontend/releases/download/%{version}/%{name}-%{version}.tar.gz
```

Comment:
Nitpick: it may be shorter / nicer to use the forge macros instead of the full URL, see https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_using_forges_hosted_revision_control

Reply:
But it may as well not work due to the mismatch of the project name and package name :shrug:

Reply:
Well whatever will upload the release to github will need to make sure to attach the artefact with the correct name.

Reply:
fwiw I just took this from cockpit

Reply:
Sure, feel free to ignore the suggestion.

Reply:
if ok I will just resolve this, can always revisit it once we start making github releases.